solo/docker/start.sh

```diff
@@ -82,6 +82,37 @@ com.spotify.helios.master.MasterMain \
 $HELIOS_MASTER_OPTS \
 &
 
-set +x
-# Sleep or execute command line
-while :; do sleep 1; done
+if [ "$HELIOS_SOLO_SUICIDE" -eq 1 ]; then
```

Comment:
I think this may cause an error if `$HELIOS_SOLO_SUICIDE` is not defined

From a quick test:

```
+ '[' '' -eq 1 ']'
/tmp/su.sh: line 3: [: : integer expression expected
```

`1` should probably be wrapped in quotes